build-logic/conventions/src/main/kotlin/com/squareup/anvil/conventions/utils.kt

```diff
@@ -0,0 +1,11 @@
+package com.squareup.anvil.conventions
+
+import org.gradle.api.Project
+import org.gradle.api.publish.PublishingExtension
+
+internal fun Project.isInMainAnvilBuild() = rootProject.name == "anvil"
+
+internal fun Project.isInDelegateBuild() = rootProject.name == "delegate"
```

Comment:
Can you add some high level documentation somewhere around the main build vs delegate build, why we have both, etc?